### PR TITLE
Improvements to contributing documentation

### PR DESCRIPTION
**Pull request by stefansedich**
Improving contribution documentation with some general cleanup and adding sections that were linked but were actually 404ing in the hope of creating a simpler contribution experience.

### CODE REVIEW

**Comment by stefansedich** [comment]
Opening up as I have had some time to sit down and write something up, ready for feedback!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # How to Contribute
 
-The Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
+The Flux Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
```
Comment (hiddeco) [inline CONTRIBUTING.md]:
```suggestion
The Helm Operator is [Apache 2.0 licensed](LICENSE) and accepts contributions
```

We only call it 'Flux Helm Operator' when additional context is required, given someone is already 'in' the Flux ecosystem I would simply omit the project branding here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,10 +16,9 @@ The idea here is that new issues are confirmed, which might require asking
 for more information, testing with a fresh environment. Once confirmed, the
 `blocked needs validation` label is removed, and the issue can be worked on.
 
-To set up Flux to test things, there's documentation about setting up a
-[standalone install](https://docs.fluxcd.io/en/latest/tutorials/get-started/)
-and a [Helm install](https://docs.fluxcd.io/en/latest/tutorials/get-started-helm/),
-which might be helpful.
+To set up Flux Helm Operator to test things, there's documentation about setting up a
```
Comment (hiddeco) [inline CONTRIBUTING.md]:
```suggestion
To set up Helm Operator to test things, there's documentation about setting up a
```

File: docs/contributing/get-started-developing.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+...
+```
+
+## Congratulations!
+
+You did it! you made your first change to the HelmRelease CRD, built, deployed and ran an instance of the Flux Helm Operator to see it in action.
+
+You should now hopefully be more comfortable with making changes and running the Flux Helm Operator locally, and be ready to tackle your [first issue](https://github.com/fluxcd/helm-operator/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+To find out more about the Flux Helm Operator community and our contribution workflow have a look at the [contributing guide](https://docs.fluxcd.io/projects/helm-operator/en/stable/contributing/introduction/).
```
Comment (hiddeco) [inline docs/contributing/get-started-developing.md]:
```suggestion
To find out more about the Flux Helm Operator community and our contribution workflow have a look at the [contributing guide](introduction.md).
```

`mkdocs` figures out relative links to other documents by itself and transforms them to the appropriate URLs :-)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -30,7 +29,7 @@ By contributing to this project you agree to the Developer Certificate of
 Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. No action from you is required, but it's a good idea to see the
-[DCO](DCO) file for details before you start contributing code to Flux.
+[DCO](DCO) file for details before you start contributing code to Flux Helm Operator.
```
Comment (hiddeco) [inline CONTRIBUTING.md]:
```suggestion
[DCO](DCO) file for details before you start contributing code to the Helm Operator.
```

Missed this one.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -73,11 +72,11 @@ This is a rough outline of how to prepare a contribution:
...
 
-Refer to the [building doc](https://docs.fluxcd.io/en/latest/contributing/building/) to
-find out how to build from source.
+Refer to the [Building](https://docs.fluxcd.io/projects/helm-operator/en/stable/contributing/building/) guide to find out how to build from
+source.
 
-Refer to the [Get Started Developing](https://docs.fluxcd.io/en/latest/contributing/get-started-developing/)
-guide for a walkthrough on developing Flux locally.
+Refer to the [Get started developing](https://docs.fluxcd.io/projects/helm-operator/en/stable/contributing/get-started-developing/) guide
+for a walkthrough on developing Flux Helm Operator locally.
```
Comment (hiddeco) [inline CONTRIBUTING.md]:
```suggestion
for a walkthrough on developing the Helm Operator locally.
```

**Review by hiddeco — APPROVED** [review]
Thank you very much @stefansedich :star: :beer:

**Pull request merged**